src/orion/algo/space.py

```diff
@@ -419,6 +424,15 @@ def __contains__(self, point):
 
         return numpy.all(point_ >= low) and numpy.all(point_ <= high)
 
+    def get_prior_string(self):
+        """Build the string corresponding to current prior"""
+        prior_string = super(Real, self).get_prior_string()
+
+        if self.precision != 4:
+            return prior_string[:-1] + f", precision={self.precision})"
```

Comment:
If the precision is explicitly set to 4 by the user it will not be added to the prior string.

Reply:
yes, that is expected

Reply:
It's not coherent. :( Maybe replace precision by a property and return 4 if None, and within get_prior_string look at _precision instead of the property.